searx/searx
# Remove HTTP connections as much as possible

## What does this PR do?

* Fix dbpedia autocomplete and use HTTPS instead of HTTP.
* Documentation & comments: remove http:// URL as much as possible (few exceptions).

## Why is this change important?

Remove unsecured connections.

## How to test this PR locally?

* check dbpedia autocomplete is fixed
* check each changed URL

## Author's checklist



## Related issues
